packages/front/src/lib/components/DefinePassword/DefinePassword.controller.js

```diff
@@ -38,7 +39,8 @@ export default class DefinePasswordController {
     }
 
     _handleValidDispatch() {
-        if (!this.showPasswordError.value && !this.showConfirmError.value) this.dispatch("valid");
+        if (!this.values.password || !this.values.confirmPwd) return this.dispatch("error");
```

Comment:
Uttiliser _handleErrorDispatch pour centralier ! et ducoup je me dit que cette fonction est pas forcement hyper bien nomée

Reply:
En regardant ça me donne plutôt envie de faire l'inverse et de mettre `this.dispatch("error") là où `_handleErrorDispatch` et de la supprimer

Reply:
Comme tu veux, j'avoue que j'ai bien centrailier les appels similiaire mais en sois ça me va !
Pour le nom de fonctiion je parlait de _handleValidDispatch

Reply:
Ben là ce serait vraiment faire un alias pour un appel d'une fonction de la classe déjà donc ça me paraîtrait juste verbeux. Ok pour renommer par contre je fais ça.

Reply:
C'était pénible de refacto les tests pour mock `this.dispatch` donc j'ai juste fait du renommage finalement :grin: